docs/governance/steering/elections.md

```diff
@@ -0,0 +1,126 @@
+
+### Eligibility for Nomination and Endorsement
+
+Any FINOS member may nominate as many people as they wish to. We recommend that you validate the eligibility of your candidate prior to nomination.
+
+[Eligible voters](#eligibility-for-voting) may endorse nominees or campaign as they see fit.
+
+## Terms and Election Cycles
+
+Steering committee members are elected to serve a two (2) year term. Members can serve two (2) consecutive terms and a lifetime of four (4) terms. Terms that result in less than or equal to one (1) year served are exempt.
+
+Election cycles are scheduled such that roughly half of the seats come up for re-election each year for purposes of continuity. The exact number of seats alternates between three (3) and four (4).
+
+### Bootstrap Election
+
+In the [bootstrap] process (which does not follow the guidelines set forth here) four (4) seats will be allocated for 364-day terms, to stagger all future elections. The committee will be responsible for self-allocation of these seats.
+
+### Emeritus Term
+
+Members of the steering committee who complete a term will graduate to becoming Emeritus members of the steering committee upon vacating their seat. This confers honor on the recipient, acknowledging the significant contributions they have made to the project. Emeritus members have no binding vote, and no expectation of continued participation in steering committee affairs.
+
+## Election Schedule and Operation
+
+After the [bootstrap] election, the steering committee is responsible for the appointment of election officers to operate the election and circulate an exact election timeline to the community. The steering committee should provide the election officer with a desired timeline based on current community needs.
+
+For example:
+
+- End of July
+  - Election officers
+  - Voter eligibility criteria
+  - Election preparation
+- September   
+  - Nomination period and election
+- October  
+  - Conclusion of Election
+  - Results announced at first community meeting after the election concludes
+
+The election officers will choose exact dates for each step and propose the final schedule to the Steering Committee.
+
+### Nomination and Election Procedure
+
+Nominations will be made using the _Nomination_ GitHub Issue template.
+
+Elections will be held using an online preference election system which supports Condorcet elections. The most preferred candidates will be elected to the open seats.
+
+### Election Officer Eligibility
+
+The steering committee should choose multiple election officers using the following criteria, so as to promote healthy rotation and diversity:
+
+- Each individual must be eligible to vote.
+- Each individual must have been a FINOS member for at least one year.
+- At least one election officer should have served before.
+- At least one election officer should have never served before.
```

Comment:
Combine numbers 83 & 84 - At least one election officer should have either served, or not served before.